src/decorator-annotator.ts

```diff
@@ -120,6 +120,11 @@ class ClassRewriter extends Rewriter {
       case ts.SyntaxKind.Decorator:
         // Skip emit of all decorators, as they are specially handled.
         return true;
+      case ts.SyntaxKind.PropertyDeclaration:
```

Comment:
Move up the the `ts.SyntaxKind.MethodDeclaration` so there is only one `case` block for all of them.